ting_infomedia.module

```diff
@@ -60,6 +71,36 @@ function ting_infomedia_theme($existing, $type, $theme, $path) {
   return $render_array;
 }
 
+/**
+ * Implements infomedia_url_alter (@see ting.entities.inc::getOnline_url
+ *
+ * @param $url
+ *   The original url.
+ * @param $entity *
+ *   The entity the url belongs to.
+ */
+function ting_infomedia_infomedia_url_alter(&$url, $entity){
+  $url = ting_infomedia_parse_url($url);
+}
+
+/**
+ * Alter infomedia url.
+ *
+ * @param $url
+ *   The original url.
+ * @return string
+ *   The modified url.
+ */
+function ting_infomedia_parse_url($url){
+  $url = parse_url($url, PHP_URL_QUERY);
+  parse_str($url, $parts);
```

Comment:
`$parts` is an undefined variable at this point. Please declare it. It will cause a notice since it is being passed by reference.